lib/kernel/match/match_chunk.ml

```diff
@@ -54,3 +54,4 @@ let to_chunks source l =
   |> List.sort ~compare:(fun (left : Range.t) right -> Int.compare left.match_start.offset right.match_start.offset)
   |> fold_matches []
   |> range_chunks source
+  |> List.rev
```

Comment:
Reverse the chunks so they are in order (side note: can you really not append / pop with ocaml lists??)

Reply:
you can append with `@`

Reply:
Huh. I tried that. It complained and stack overflow was pessimistic